packages/features/portfolio/src/containers/portfolio-overview/containers/portfolio-holdings-overview.tsx

```diff
@@ -35,22 +37,22 @@ const PortfolioHoldingsOverview = ({ holdings }: IProps) => {
                   borderBottomStyle: 'solid',
                 }}
               >
-                <Typography
-                  variant="h3"
+                <H3
                   sx={{
```

Comment:
Remove most of the sx settings here and just leave the text transform. I want all the font sizes set using the notional-theme provider